source/manual/architecture.html.md

```diff
@@ -4,12 +4,12 @@ title: Architectural overview of GOV.UK applications
 section: Basics
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2018-07-03
+last_reviewed_on: 2018-10-02
 review_in: 3 months
 ---
 
-![](/manual/images/govuk-architecture.jpg)
+![](https://www.draw.io/?lightbox=1&highlight=0000ff&layers=1&nav=1#G1qTEpv2kCzghqZpUF86UyQj4o0dZ97gTm)
```

Comment:
This doesn't seem to work?

https://govuk-developer-docs-pr-1270.herokuapp.com/manual/architecture.html